src/components/PagesComponent/PeopleFirst/style.js

```diff
@@ -76,10 +80,16 @@ const PeopleFirstWrapper = styled.div`
      }
      &:nth-child(5){
       border-bottom:  1px solid ${({ theme }) => theme.colors.dividerColor};
+      ${media.sm`
+      border-bottom:  1px solid ${({ theme }) => theme.colors.dividerColor} !important;
+      `}
      }
      &:nth-child(6){
       border-right: 0;
       border-bottom:  1px solid ${({ theme }) => theme.colors.dividerColor};
+      ${media.sm`
+      border-bottom:  1px solid ${({ theme }) => theme.colors.dividerColor} !important;
```

Comment:
same here

Reply:
brother in responsive border not vissible check staging